Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp

```diff
@@ -398,21 +398,11 @@ double WKBundlePageGetTextZoomFactor(WKBundlePageRef pageRef)
     return WebKit::toImpl(pageRef)->textZoomFactor();
 }
 
-void WKBundlePageSetTextZoomFactor(WKBundlePageRef pageRef, double zoomFactor)
-{
-    WebKit::toImpl(pageRef)->setTextZoomFactor(zoomFactor);
-}
-
 double WKBundlePageGetPageZoomFactor(WKBundlePageRef pageRef)
```

Comment:
Let's remove these accessors too if possible.

Reply:
I can do this, but it doesn’t seem necessary to send another IPC message to the UI process to get the current zoom. The zoom factor should be the same in the main frame process, and we need this API to get that.